File: 2310817120009_AlysaArmelia_PemrogramanI_Modul6/Python/PRAK603-2310817120009-AlysaArmelia.py
n1, n2 = map(int, input().split())

if (n1 != n2):

    print ("Jumlah tidak sama")
    exit(1)

list_n1 = list(map(int, input().split()))
list_n2 = list(map(int, input().split()))

if (len(list_n1) == n1 and len (list_n2) == n2):

    tik = 0

    while (tik < n1):
        keseluruhan = list_n1[tik] * list_n2[tik]
        print(keseluruhan, end=" ")
        tik+=1

File: 2310817120009_AlysaArmelia_PemrogramanI_Modul6/Bahasa C/PRAK603-2310817120009-AlysaArmelia.c
#include <stdio.h>

int main() 
{
    int tik;
    int matematika; 
    int n1;
    int n2;

    int keseluruhan[45] = {1};

    scanf("%d %d", &n1, &n2);

    int Limit [n1][n2];

    if (n1 == n2) 
    {

        for (tik = 0; tik < 2; ++tik) 
        {
            for (matematika = 0; matematika < n2; ++matematika) 
            {
                scanf("%d", &Limit[tik][matematika]);
            }
        }
        
        for (tik = 0; tik < 2; ++tik) 
        {
            for (matematika = 0; matematika < n2; ++matematika)
            {
                keseluruhan[matematika] = Limit[0][matematika] * Limit[1][matematika];
            }
        }

        for (matematika = 0; matematika < n2; ++matematika) 
        {
            printf("%d ", keseluruhan[matematika]);
        }
        
    } else {
        
        printf("Jumlah tidak sama\n");

    }
return 0;
}